package.json

```diff
@@ -45,7 +45,10 @@
     "typescript": "^4.9.3"
   },
   "dependencies": {
+    "@types/bytebuffer": "^5.0.44",
+    "@types/ramda": "^0.28.20",
     "blake-hash": "^2.0.0",
+    "bytebuffer": "^5.0.1",
```

Comment:
if you are using bytebuffer only for tests, please add this package and @types/bytebuffer into devDependencies of package.json, not direct dependencies.

Reply:
Done. [link to commit](https://github.com/iden3/js-crypto/pull/2/commits/5b5bf5f7cb363f86ddde79ea2313b3d89bb8bf93)